src/dnx.common/utils.cpp

```diff
@@ -114,5 +114,44 @@ namespace dnx
                 ? path.substr(0, last_separator)
                 : path;
         }
+
+#if defined (_WIN32)
+        const wchar_t* get_windows_version()
+        {
+            OSVERSIONINFO version_info;
+            ZeroMemory(&version_info, sizeof(OSVERSIONINFO));
+            version_info.dwOSVersionInfoSize = sizeof(OSVERSIONINFO);
+
+#pragma warning(disable:4996)
+            GetVersionEx(&version_info);
+#pragma warning(default:4996)
+
+            if (version_info.dwMajorVersion == 10)
+            {
+                return L"10";
+            }
+
+            if (version_info.dwMajorVersion == 6)
+            {
+                if (version_info.dwMinorVersion == 3)
+                {
+                    return L"81";
```

Comment:
Can we reformat this into a full version number? "8.1", "10.0", etc. for example. I realize not all OSes will fit this criteria but I'd generally like the version numbers to be parsable by System.Version, etc.

If we need to shrink it back down to "81", we can do it after having parsed it.